var/spack/repos/builtin/packages/libfabric/package.py

```diff
@@ -12,16 +12,13 @@ class Libfabric(AutotoolsPackage):
        fabric communication services to applications."""
 
     homepage = "https://libfabric.org/"
-    url      = "https://github.com/ofiwg/libfabric/releases/download/v1.6.1/libfabric-1.6.1.tar.gz"
```

Comment:
Keep this here (and update to one of the newer tar.bz2 versions) so that `spack versions libfabric` and `spack checksum libfabric` still work.